### DISCUSSION TITLE
Choose location instead of importing?

### DISCUSSION THREAD

**Discussion by Psi25** [carson-katri/dream-textures#582] [Q&A] [answered]
How do I choose model location instead of importing? I don't what my main drive to fill up with models.

**Comment by carson-katri** [comment, marked as answer]
The addon does not read ckpt files, so they must be imported which converts them to the proper format.

**Reply by Ainaemaet** [reply]
In that case, the question would be "can you please make it so that we can import the models to a location of our choosing?". 

As it is, most of the people I know are running smaller SSD's as their system drive, so having the import location default to system drive is somewhat problematic. 

Hope I don't sound rude in my submissions here, I'm somewhat neuro-divergent and doing my best to communicate both concisely and politely! :)
